source/fundamentals/data-formats/documents.txt

```diff
@@ -14,34 +14,351 @@ Documents
 Overview
 --------
 
-- Definition - what is a document, what format is it, where to find more info 
-- Table comparison of classes (columns for: implements Map, provides type safety, is an interface or class, compatibility with server versions
-- Recommend Document for general use and why.
-- Duplicate field name admonition (I forgot what this is)
+A MongoDB **document** is a data structure that contains key/value pairs in
+binary JSON (BSON) format and is the base unit in the MongoDB data hierarchy.
+
+Each **document** belongs to a single **collection** and each **collection**
+belongs to a single **database**. For more information, see our MongoDB manual
```

Comment:
Thanks for bringing this up.

1) I think I could replace this with `:term:` (see Glossary terms in https://docs.mongodb.com/meta/style-guide/style/text-formatting/).
Italics could be used for introducing new terms (see "New terms" in https://docs.mongodb.com/meta/style-guide/style/text-formatting/ ) explained immediately afterwards

2) I did think about creating a graphic and searched through server/manual documentation and wasn't able to find anything suitable to link to or re-use. Maybe I can't find the right page which probably means our users can't find it either. Instead of attempting to explain it from scratch, I just wanted to introduce repetition of terms and what they are as a learning aid.

That said, if this was a language-specific concept, I would probably devote a section for it.

Reply:
I think this is a perfect opportunity for creating a glossary, so definitely think about that! Just check to make sure it's supported properly by nextgen first :)

I hoped that the linked server page would include such a graphic, too, but sadly it doesn't. Might be a good idea to ask in the #docs chat to see if anybody knows of such a graphic (or would be willing to create one for you, if they have a LucidChart license and some degree of artistic skills)

Reply:
+1 to repetition as a learning aid, I think that works quite well here.

Reply:
Didn't realize you need a glossary to use `:term:`. I removed formatting for the time being. I'm a little confused by the inconsistent formatting here, so probably something to address sometime later: https://github.com/mongodb/docs-java/blob/master/source/fundamentals/databases-collections.txt